### DISCUSSION TITLE
shelly 1 acceso / spento .... aperto/chiuso

### DISCUSSION THREAD

**Discussion by RobyBS** [indomus/forum#1664] [HUB: Home Assistant]
scusate l'ignoranza .. ma è la prima volta che uso uno shelly 1 , collegato ad un telecomando esistente , per aprire o chiudere la basculante del garage ... 

tutto funziona perfettamente , solo che per far funzionare il comando tramite google assistant devo pronunciare "hey googole , ACCENDI porta garage" ... invece che " hey google, APRI porta garage" ... la cosa fa sorridere lo so ahahhah ... 

come posso fare per far accettare all'assistente vocale il comando apri invece che accendi ? 

grazie !

**Comment by Scialla** [comment]
Su Alexa si crea una routine con il comando che vuoi, su Google sarà similare credo...

**Reply by RobyBS** [reply]
provo a vedere ... ;) ... 
per adesso grazie ! ... 


ho aggiunto le routine per apri e chiudi .. stasera quando arrivo a casa , provo e confermo che tutto funzioni .... 👍

**Reply by RobyBS** [reply]
perfetto ! .. tutto funzionante ! ... 
grazie 1000 per la dritta ! ... 
👍 👍 👍
